bump.sh

```diff
@@ -13,21 +13,4 @@
 #
 # --------------------------------------------------------------------------------------------------
 set -euo pipefail
-version=${1:-minor}
-
-echo "Starting ${version} version bump"
-
-# /bin/bash ./install.sh
-
-# Generate CHANGELOG and create a commit (see .versionrc.json)
-npx standard-version --release-as ${version}
-
-# I am sorry.
-#
-# I've gone diving through the code of `conventional-changelog` to see if there
-# was a way to configure the string and ultimately I decided that a 'sed' was the simpler
-# way to go.
-sed -i.tmp -e 's/BREAKING CHANGES$/BREAKING CHANGES TO EXPERIMENTAL FEATURES/' CHANGELOG.md
-rm CHANGELOG.md.tmp
-git add CHANGELOG.md
-git commit --amend --no-edit
+./scripts/bump.js ${1:-minor}
```

Comment:
Probably not needed since you already have a default in `bump.js`.

```suggestion
./scripts/bump.js
```

Reply:
But we still want to allow people to specify the bump type.

Reply:
You're right. Not sure why I said that. Ignore.